docs/roadmap.md

```diff
@@ -2,21 +2,21 @@
 
 This document provides a high-level view on our product roadmap for this and upcoming quarters. It also lists roadmap items of past quarters.
 
-## Q1 2022
+## Q2 2022
 
-### Dependant Resources
+### More flexible Resource Types
 
-Some resources require inputs form other resources to be provisioned. For example, an "RDS Instance for MySQL" is needed in order to provision a `mysql` database or an "Azure Service Bus Namespace" might be needed in order to provision an `azure-service-bus-topic`. For more sophisticated setups, it might be necessary to provision underlying infrastructure which other resources will then depend on. It might also be necessary to inject common inputs into multiple resources (e.g. individual subscriptions for multiple workloads must all subscribe to the same topic). The feature will allow resource definitions to depend on resources defined by other resource definitions.
+Humanitec already provides a range of core resource types such as [`dns`](https://docs.humanitec.com/integrations/resource-types/dns), [`postgres`](https://docs.humanitec.com/integrations/resource-types/postgres) and [`volume`](https://docs.humanitec.com/integrations/resource-types/volume). Since the introduction of Resource References (previously known as Dependent Resources) a lot more can be achieved with the resources system. We will be introducing a number of new types including `tsl-cert` for modelling TLS Certificates, `k8s-namespace` to make it easier to programmatically defined the name of namespaces and additional types to model dependencies on services in other applications or hosted in other platforms.
 
-### App Details Page UI Update
+### Virtual Drivers
 
-Following several months of user testing and customer feedback we are planning on introducing a number of changes to improve the usability, predictability and scalability of our UI. The focus lies on the App Details Page, the control centre of an App. We want to address issues related to interaction design, information hierarchy, status communication as well as aesthetics while touching on UI patterns which are likely to change the look and feel of the Humanitec UI as a whole.
+The introduction of generic drivers such as the [`humanitec/template`](https://docs.humanitec.com/integrations/resource-drivers/template) driver add a lot of power to the resource system. Unfortunately, using these drivers multiple times requires the driver inputs to be duplicated. This makes it hard to maintain and understand these [Resource Definitions](https://docs.humanitec.com/reference/concepts/resources/definitions). Virtual Drivers will provide a way of encapsulating and reusing the inputs of a driver across multiple [Resource Definitions](https://docs.humanitec.com/reference/concepts/resources/definitions).
```

Comment:
"resource system" vs. "resources system" (see below "More flexible Resource Types)

Reply:
This is not actually a typo!